evalcache.plugin.zsh

```diff
@@ -7,7 +7,8 @@
 export ZSH_EVALCACHE_DIR=${ZSH_EVALCACHE_DIR:-"$HOME/.zsh-evalcache"}
 
 function _evalcache () {
-  local cacheFile="$ZSH_EVALCACHE_DIR/init-$1.sh"
+  BINARY=$1
+  local cacheFile="$ZSH_EVALCACHE_DIR/init-${BINARY##*/}.sh"
```

Comment:
Just making sure I understand this correctly: `${foo##pat}` removes the longest matching pattern, so this just extracts the trailing filename in all cases?

Reply:
Yep, that's exactly what it does